artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -4457,6 +4457,15 @@
             </xsd:annotation>
          </xsd:element>
 
+         <xsd:element name="intermediate-message-buffer-initial-size" type="xsd:int" maxOccurs="1" minOccurs="0">
```

Comment:
Specify the default value here as with the other elements.

Reply:
This is done.